src/chigame/api/tests.py

```diff
@@ -0,0 +1,117 @@
+from chigame.games.models import Game
+
+
+class GameTests(APITestCase):
+    def test_create_game(self):
+        """
+        Ensure we can create a new game object.
+        """
+
+        # Create a game object
+        url = reverse("game-list")
+        data = {"name": "The Witcher 3", "description": "Geralt of Rivia", "min_players": 1, "max_players": 4}
+        response = self.client.post(url, data, format="json")
+        # Check that the game object was created correctly
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.assertEqual(Game.objects.count(), 1)
+        self.assertEqual(Game.objects.get().name, "The Witcher 3")
+        self.assertEqual(Game.objects.get().description, "Geralt of Rivia")
+        self.assertEqual(Game.objects.get().min_players, 1)
+        self.assertEqual(Game.objects.get().max_players, 4)
+
+    def test_get_game(self):
+        """
+        Ensure we can get a game object.
+        """
+
+        # Create a game object
+        url = reverse("game-list")
+        data = {"name": "The Witcher 3", "description": "Geralt of Rivia", "min_players": 1, "max_players": 4}
+        response = self.client.post(url, data, format="json")
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        # Get the game object
+        url = reverse("game-detail", kwargs={"pk": 1})
+        response = self.client.get(url, format="json")
+
+        # Check that the game object was retrieved correctly
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(Game.objects.count(), 1)
+        self.assertEqual(Game.objects.get().name, "The Witcher 3")
+        self.assertEqual(Game.objects.get().description, "Geralt of Rivia")
+        self.assertEqual(Game.objects.get().min_players, 1)
+        self.assertEqual(Game.objects.get().max_players, 4)
+
+    def test_update_game(self):
+        """
+        Ensure we can update a game object.
+        """
+
+        # Create a game object
+        url = reverse("game-list")
+        data = {"name": "The Witcher 3", "description": "Geralt of Rivia", "min_players": 1, "max_players": 4}
+        response = self.client.post(url, data, format="json")
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        # Update the game object
+        url = reverse("game-detail", kwargs={"pk": 1})
+        data = {"name": "The Witcher 3: Wild Hunt", "max_players": 5}
+        response = self.client.patch(url, data, format="json")
+
+        # Check that the game object was updated correctly
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(Game.objects.count(), 1)
+        self.assertEqual(Game.objects.get().name, "The Witcher 3: Wild Hunt")
+        self.assertEqual(Game.objects.get().description, "Geralt of Rivia")
+        self.assertEqual(Game.objects.get().min_players, 1)
+        self.assertEqual(Game.objects.get().max_players, 5)
```

Comment:
Given that you're doing these kind of checks a lot, it might be useful to consider a helper function which can test some given fields to see if they have some associated values. 

That way you could pass in a more compact structure like a dictionary of expected fields and values, then that dictionary could be fed to your helper function which generically checks the provided fields via `Game.objects.get().[field_name]`.